backend/src/test/java/de/neuefische/backend/company/CompanyControllerIntegrationTest.java

```diff
@@ -142,4 +147,34 @@ void addCompany_whenInvalidRequest_thenReturnBadRequest() throws Exception {
                 .andExpect(MockMvcResultMatchers.status().isBadRequest());
     }
 
+    @DirtiesContext
+    @Test
+    void deleteCompany_whenCompanyInDB_thenDBDoesNotContainCompanyAnymore() throws Exception {
+
+        MvcResult resultCompanyToDelete = mockMvc.perform(MockMvcRequestBuilders.post("/api/company")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content("""
+                            {
+                                              "name": "TestFirma3",
+                                              "country": "TestCountry3",
+                                              "city": "TestCity3",
+                                              "plz": "133335",
+                                              "street": "TestStreet",
+                                              "streetNumber": "3",
+                                              "phoneNumber": "123-3333-7890",
+                                              "email": "test3@example.com",
+                                              "website": "https://www.testfirma3.com",
+                                              "comment": "This is a test comment333."
+                            }
+                        """)).andReturn();
+
+
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
Use dependency injection for objectMapper

`@Autowired
ObjectMapper objectMapper;`